### ISSUE TITLE
Snippets are not indented correctly after insertion

### ISSUE THREAD

**Issue by manasthakur** [KeyboardFire/vim-minisnip#9]
I read about the plugin on r/vim and I like the minimalism. There is an issue though.

Suppose I created the following snippet in a file `_java_if`:

```
if ({{++}}) {
    {{++}}
}
```

Now when I expand the snippet in a java file, only the first line (`if...`) is indented properly. The rest of the text starts from the beginning of the line, which is unexpected.

Am I missing something?

**Comment by joereynolds** [comment]
This seems to happen to me too :(

![image](https://cloud.githubusercontent.com/assets/9536682/26717379/16aeecb2-4775-11e7-8b7b-e8058f5cbf8d.png)

**Comment by manasthakur** [comment]
@LemonBoy's [PR](https://github.com/KeyboardFire/vim-minisnip/pull/10) seems to solve the issue for me. However, seeing the time since it's pending, it seems @KeyboardFire is busy now-a-days.

**Comment by joereynolds** [comment]
Yep same, I've forked and merged all backwards compatible PR's so far.
It's a shame that the PR's haven't been accepted (yet) :(

**Comment by joereynolds** [comment]
For anyone else reading, I have a [fork](https://github.com/joereynolds/vim-minisnip) which fixes this (and a few other) bug.

**Comment by partounian** [comment]
Hey @joereynolds, thanks for your work! I suggest you make a new issue mentioning you have merged in new PRs/are maintaining this.

**Comment by joereynolds** [comment]
@partounian consider it done
